crates/ark/src/lsp/completions/completion_context.rs

```diff
@@ -13,12 +13,14 @@ use crate::lsp::completions::sources::composite::pipe::find_pipe_root;
 use crate::lsp::completions::sources::composite::pipe::PipeRoot;
 use crate::lsp::document_context::DocumentContext;
 use crate::lsp::state::WorldState;
+use crate::treesitter::node_find_containing_call;
```

Comment:
New utility that allows me to eliminate 3 instances of same logic in the call, custom, and pipe completion sources.